packages/@aws-containers/aws-ecs-builder/lib/extensions/appmesh.ts

```diff
@@ -0,0 +1,326 @@
+import * as appmesh from '@aws-cdk/aws-appmesh';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as ecr from '@aws-cdk/aws-ecr';
+import * as ecs from '@aws-cdk/aws-ecs';
+import * as iam from '@aws-cdk/aws-iam';
+import * as cdk from '@aws-cdk/core';
+import { Service } from '../service';
+import { Container } from './container';
+import { ServiceExtension, ServiceBuild } from './extension-interfaces';
+
+// A map of AWS account ID's which hold the App Mesh image in various
+// regions
+enum APPMESH_ECR_ACCOUNTS {
```

Comment:
I believe this info belongs in the `region-info` package. At least in the `appmesh` package - I don't think this is the greatest place for it.

Reply:
We have some work on the underlying L2 appmesh package scheduled. I'll discuss with the team when that second pass begins. I agree this is something we should probably implement as part of that pass to catch the L2's up to the current CFN features.

Reply:
Agree, `appmesh` package can own this configuration and we can tackle it as part of L2 work (either myself or @dfezzie). We're considering an SSM parameter for this longer term (aws/aws-app-mesh-roadmap#257).